app/assets/javascripts/admin/templates/api-keys-index.hbs

```diff
@@ -19,7 +19,7 @@
       <tr class={{if k.revoked_at "revoked"}}>
         <td class="key">
           {{#if k.revoked_at}}{{d-icon 'times-circle'}}{{/if}}
-          {{k.shortKey}}
+          {{k.truncated_key}}...
```

Comment:
Guess it could be nice to have this in api-key model:

```
import { fmt } from "discourse/lib/computed";

truncatedKey: fmt("truncated_key", "%@...")
```

It's minor but, it avoid repeating the `...` each time we use it, and I like the idea of limit as much as possible snake_case in `hbs` files, obviously we will always have some...